src/CoreWCF.Primitives/src/CoreWCF/Security/CanonicalFormWriter.cs

```diff
@@ -0,0 +1,82 @@
+
+
+
+using System.IO;
+using System.Text;
+
+namespace CoreWCF.IdentityModel
+{
+    internal abstract class CanonicalFormWriter
+    {
+        // internal static readonly UTF8Encoding Utf8WithoutPreamble = new UTF8Encoding(encoderShouldEmitUTF8Identifier = false);
+        internal static readonly Encoding Utf8WithoutPreamble = Encoding.UTF8;
```

Comment:
It looks like you copied this from the WCF client. This looks like it's a bug as Encoding.UTF8 is created with encoderShouldEmitUTF8Identitifer defaults to true. Use the commented out code instead.